ui/components/application/BillingAddress.tsx

```diff
@@ -19,10 +24,12 @@ const BillingAddress = ({ register, errors }: Props): JSX.Element | null => {
         {t("application:Page3.subHeading.billingAddress")}
       </FormSubHeading>
       <TextInput
-        ref={register({ required: true, maxLength: 255 })}
+        {...register("billingAddress.streetAddress", {
```

Comment:
Eikös noi optionssit sais määriteltyä suoraa schemaann? vois vaan laittaa {...register("muuttujanimi")}

Reply:
Yeah, that's a great idea. however there's currently no validation library in use in this fe. I created a separate ticket for doing this.